tests/testData/userTestData.js

```diff
@@ -93,3 +95,109 @@ export const validUsernameQueryEthan = {
   creationTime: "2024-02-27T00:00:00.000Z",
   modificationTime: "2024-02-27T00:00:00.000Z",
 };
+
+export const userToApprove1 = {
```

Comment:
As per above, these can be added in seeding rather than in testing data. It would be wise to make 'seeding ready' users, with actual usernames, and other fields. In the bio, or as a comment in seeding, you can note that this set of users will be used for particular endpoint testing.